### DISCUSSION TITLE
Question about request processing

### DISCUSSION THREAD

**Discussion by eugenestepaniuk** [sequelize/sequelize#16061] [Help & Questions]
hello, could someone help me, I have a nodejs server in which I use Sequelize to work with the database, when I execute a query on the server to the database, for example, to delete a row from a table, it is not always deleted from the first query, the query itself is executed, but the row in the database table is not deleted, but it can be deleted after the second or third query execution, what could be the reason?
I use version 6.30.0, my Sequelize connection:
```
export default new Sequelize({
  dialect: "mssql",
  database: environment.database.database,
  host: environment.database.host,
  port: environment.database.port,
  username: environment.database.username,
  password: environment.database.password,
  logging: !environment.production ? console.log : false,
  dialectOptions: {
    options: {
      enableArithAbort: true,
      cryptoCredentialsDetails: {
        minVersion: "TLSv1",
      },
    },
  },
  define: {
    freezeTableName: true,
    timestamps: false,
  },
});
```

**Comment by eugenestepaniuk** [comment]
Could you help me in this matter?

**Comment by WikiRik** [comment]
What queries are being executed by sequelize?

**Comment by eugenestepaniuk** [comment]
absolutely any, getting, adding, editing, deleting data from the database

**Reply by ephys** [reply]
If you can, verify that the queries reach the database

**Reply by eugenestepaniuk** [reply]
yes, the request itself is being executed on the server client
![image](https://github.com/sequelize/sequelize/assets/48010641/63ea1a13-74b7-4930-bc9e-ccafc12ac81f)

but the row in the table is not deleted

**Comment by eugenestepaniuk** [comment]
What do you think about my last answer?
And one more question arose, if I execute one query to the database when rendering the page, then the query takes about 20-30 ms, but if I execute two or more queries, they are already running much longer, 2000-3000-5000 ms . Asynchrony is present on both the client and the server, what could be the problem?
